exercises/practice/state-of-tic-tac-toe/StateOfTicTacToe.tests.ps1

```diff
@@ -0,0 +1,275 @@
+BeforeAll {
+    . "./StateOfTicTacToe.ps1"
+}
+
+Describe "StateOfTicTacToe test cases" {
+    Context "won games" {
+        It "Finished game where X won via left column victory" {
+            $grid  = @( "XOO",
+                        "X  ",
+                        "X  ")
+            $got    = Get-StateOfTicTacToe -Board $grid
+            $want   = [GameStatus]::WIN
+
+            $got | Should -BeExactly $want
+        }
+
+        It "Finished game where X won via middle column victory" {
+            $grid  = @( "OXO",
```

Comment:
The spacing seems to be a bit off, it uses 2 but to be on the same level as the other one it has to be 3 or if not use 1 spacing I would say